labelImg.py

```diff
@@ -1573,23 +1600,25 @@ def read(filename, default=None):
         return default
 
 
-def get_main_app(argv=[]):
+def get_main_app(argv=None):
```

Comment:
Rationale: Function arguments should not be mutable. This is the preferred solution.